routing/message/handlers/passthrough/telemetry_handler.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2022 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
+package passthrough
+
+import (
+	"strings"
+
+	"github.com/ThreeDotsLabs/watermill"
+	"github.com/ThreeDotsLabs/watermill/message"
+	"github.com/eclipse-kanto/azure-connector/config"
+	"github.com/eclipse-kanto/azure-connector/routing"
+	"github.com/eclipse-kanto/azure-connector/routing/message/handlers"
+	"github.com/eclipse-kanto/suite-connector/connector"
+)
+
+const (
+	telemetryHandlerName = "passthrough_telemetry_handler"
+)
+
+// telemetryHandler forwards incoming device messages published on certain local topics to the Azure Iot Hub.
+type telemetryHandler struct {
+	connSettings *config.AzureConnectionSettings
+	topics       []string
+}
+
+// CreateTelemetryHandler instantiates a new passthrough telemetry handler that forward messages received from local message broker on the given topics as device-to-cloud messages to Azure IoT Hub.
+func CreateTelemetryHandler() handlers.MessageHandler {
+	return &telemetryHandler{}
+}
+
+// Init gets the list of local topics and also the device ID that is needed for the message forwarding.
+func (h *telemetryHandler) Init(settings *config.AzureSettings, connSettings *config.AzureConnectionSettings) error {
+	h.connSettings = connSettings
+	h.topics = strings.Split(settings.PassthroughTelemetryTopics, ",")
```

Comment:
I am curious whether we have to do some validations of the given topics?

Reply:
I added some extra messages to have error logs in case a handler is not OK.
For any fancier validations, I propose to do it as part of [#21].